### PR TITLE
Generate google-cloud-container v1

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

**Comment by quartzmo** [comment]
Regarding branding, should "Container Engine" still be used in documentation, or should all usages now be "Kubernetes Engine"? (I see both in this PR.)

File: google-cloud-container/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### Next Steps
+- Read the [Client Library Documentation][] for Google Container Engine API
+  to see other available methods on the client.
+- Read the [Google Container Engine API Product documentation][Product Documentation]
+  to learn more about the product and see How-to Guides.
+- View this [repository's main README](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/README.md)
+  to see the full list of Cloud APIs that we cover.
+
+[Client Library Documentation]: https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-container/latest/google/container/v1
+[Product Documentation]: https://cloud.google.com/container-engine
```
Comment (quartzmo) [inline google-cloud-container/README.md]: This URL now redirects to: `https://cloud.google.com/kubernetes-engine/`

File: google-cloud-container/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Ruby Client for Google Container Engine API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+
+[Google Container Engine API][Product Documentation]:
```
Comment (quartzmo) [inline google-cloud-container/README.md]: The title of this page is now "Google Kubernetes Engine".

**Comment by quartzmo** [comment]
Going beyond documentation, should the name of this package be changed from `google-cloud-container` to `google-cloud-kubernetes`?

**Comment by geigerj** [comment]
@quartzmo The API team has requested that we retain the autogen `google-cloud-container`, `Google Container Engine` references, etc., *even though* it's inconsistent with cloud.google.com.

**Comment by geigerj** [comment]
I'm surprised by this, but it was an explicit decision.

File: google-cloud-container/acceptance/google/cloud/container/v1/cluster_manager_smoke_test.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+require "minitest/autorun"
+require "minitest/spec"
+
+require "google/cloud/container"
+
+describe "ClusterManagerSmokeTest" do
+  it "runs one smoke test with list_clusters" do
+    unless ENV["CONTAINER_TEST_PROJECT"]
+      fail "Usage: CONTAINER_TEST_PROJECT=<project_id> ruby #{$0}"
```
Comment (quartzmo) [inline google-cloud-container/acceptance/google/cloud/container/v1/cluster_manager_smoke_test.rb]: Why is it necessary to require that `CONTAINER_TEST_PROJECT` be set?

In every package (including this one), the Rakefile `acceptance` task also accepts the more general `GCLOUD_TEST_PROJECT`:

```ruby
project ||= ENV["GCLOUD_TEST_PROJECT"] || ENV["CONTAINER_TEST_PROJECT"]
```

**Comment by quartzmo** [comment]
Running `$ CONTAINER_TEST_PROJECT=$GCLOUD_TEST_PROJECT bundle exec rake acceptance` passes for me locally.

File: google-cloud-container/acceptance/google/cloud/container/v1/cluster_manager_smoke_test.rb (hunk 3 above)
Comment (geigerj) [reply google-cloud-container/acceptance/google/cloud/container/v1/cluster_manager_smoke_test.rb]: In this particular case it's not necessary, but part of the generator that creates the Rakefile is not currently smart enough to know that the smoke test doesn't need to read this variable.

I think we can probably just remove this check from the generator and let smoke tests that do require the env var to fail at runtime.

**Pull request merged**